src/Model.php

```diff
@@ -459,28 +516,31 @@ public function addFields($fields = [], $defaults = [])
         return $this;
     }
 
+    /**
+     * Remove field that was added previously.
+     *
+     * @param $name
+     *
+     * @throws \atk4\core\Exception
+     */
+    public function removeField($name)
+    {
+        return $this->_removeFromCollection($name, 'fields');
```

Comment:
no need for `return` here because it will not return anything anyway.
or return $this to be able to chain these methods (and write that in method comments above too `@return $this`)